reward/README.md

```diff
@@ -129,3 +129,13 @@ Using this reward fucntion, you need to create Tldr account at [Tldr Web site](h
 You also need to install openbabel. Click [here](#here) for instllation instructions.
 
 Strainfilter is licensed by the [Irwin](https://irwinlab.compbio.ucsf.edu/) and [Shoichet](https://bkslab.org/) Laboratories in the Department of Pharmaceutical Chemistry at the University of California, San Francisco (UCSF).
+
+### gnina_interaction_reward.py
+
+This reward function assesses the interactions between the receptor and the generated molecules using [ProLIF](https://doi.org/10.1186/s13321-021-00548-6).
+Need to install prolif and MDAnalysis.
```

Comment:
Add descriptions about how to specify the residue, interaction_type, cutoff, and etc.

e.g.,
residue: [residue_name][number].[chain]

Reply:
Added parameter descriptions in b91f91c4caa652403aac80f2482c458c021d5417.